Take into account the CFLAGS/LDFLAGS passed in the environment.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>

Index: host-cloop-2.634-1/Makefile
===================================================================
--- host-cloop-2.634-1.orig/Makefile	2010-12-16 09:17:52.826555645 +0100
+++ host-cloop-2.634-1/Makefile	2010-12-16 09:18:57.208713338 +0100
@@ -15,7 +15,7 @@
 endif
 
 ifdef APPSONLY
-CFLAGS:=-Wall -Wstrict-prototypes -Wno-trigraphs -O2 -s -I. -fno-strict-aliasing -fno-common -fomit-frame-pointer 
+CLOOP_CFLAGS:=-Wall -Wstrict-prototypes -Wno-trigraphs -O2 -s -I. -fno-strict-aliasing -fno-common -fomit-frame-pointer 
 endif
 
 KERNOBJ:=compressed_loop.o
@@ -53,10 +53,10 @@
 	( cd advancecomp-1.15 ; ./configure && $(MAKE) advfs )
 
 extract_compressed_fs: extract_compressed_fs.c
-	$(CC) -Wall -O2 -s -o $@ $< -lz
+	$(CC) $(CFLAGS) $(CLOOP_CFLAGS) $(LDFLAGS) -Wall -O2 -s -o $@ $< -lz
 
 cloop_suspend: cloop_suspend.o
-	$(CC) -o $@ $<
+	$(CC) $(CFLAGS) $(CLOOP_CFLAGS) $(LDFLAGS) -o $@ $<
 
 clean:
 	rm -rf create_compressed_fs extract_compressed_fs zoom *.o *.ko Module.symvers .cloop* .compressed_loop.* .tmp*
